templates/post.html

```diff
@@ -121,7 +121,12 @@ <h1 class="post_title">
 			<div class="post_footer">
 				<ul id="post_links">
 					<li><a href="{{ post.permalink }}">permalink</a></li>
-					<li><a href="https://reddit.com{{ post.permalink }}" rel="nofollow">reddit</a></li>
+					<li>
+						<a id="reddit_link" {% if prefs.visit_reddit_confirmation != "on" %}href="#popup"{% else %}href="https://reddit.com{{ post.permalink }}" rel="nofollow"{{ url }}"{% endif %}>reddit</a>
+						{% if prefs.visit_reddit_confirmation != "on" %}
+							{% call utils::visit_reddit_confirmation(post.permalink) %}
```

Comment:
@Daniel-Valentine: Make sure to remove `utils::` namespace in macro call when this goes into `templates/utils.html`.

Reply:
will keep that in mind